packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -30,6 +30,15 @@ const propTypes = {
   itemCountPerPage: PropTypes.number.isRequired,
 };
 
+const contextTypes = {
```

Comment:
Now that we are moving away from using legacy context API for i18n, could we leverage this PR to make changes using `<FormattedMessage>` instead?

Reply:
Great idea, let’s get that included now while we wait for translations to come back.

Reply:
Great idea, let’s get that included now while we wait for translations to come back.

Reply:
Resolved with https://github.com/cerner/terra-core/pull/2485/commits/2a5a237ebaf0711982686f79106f261f1317e2de